spdx/package.py

```diff
@@ -248,13 +265,14 @@ def validate_str_fields(self, fields, optional, messages):
         return messages
 
     def validate_checksum(self, messages):
-        if not isinstance(self.check_sum, checksum.Algorithm):
-            messages = messages + [
-                "Package checksum must be instance of spdx.checksum.Algorithm"
-            ]
-        else:
-            if self.check_sum.identifier != "SHA1":
-                messages = messages + ["File checksum algorithm must be SHA1"]
+        if self.check_sum is not None: # optional
```

Comment:
Can you avoid end of line comments?... also I am not sure what this optional means here?
And why not using instead:
```suggestion
        if self.check_sum:
```

Reply:
sorry for the inline comment :)  optional means that check_sum tag, according to the specs, is optional (not mandatory). But it is already written in Package class' help text, so it is not necessary to repeat it here.

As for keeping "is not None", the same reasoning applies as before for verif_code

```suggestion
        if self.check_sum is not None:
```